src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,7 @@
+.catalog {
+  width: calcCatalogWidth($card-width, $card-margin-right);
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0 auto;
```

Comment:
![image](https://user-images.githubusercontent.com/99331793/201091786-9a1b569c-f8c1-4441-b183-6c70cdc88b6a.png)

Reply:
Do not set width directly to BEM block